stack.yaml

```diff
@@ -1,4 +1,8 @@
+<<<<<<< HEAD
 resolver: lts-15.0
+=======
+resolver: lts-14.27
+>>>>>>> master
```

Comment:
I think git has messed with you a bit 😆

Reply:
That's why VS Code was asking me so many times "do you really want to commit?"